CODEOWNERS

```
@@ -2,3 +2,8 @@
 
 PublicAPI.Shipped.txt @nohwnd @MarcoRossignoli
 PublicAPI.Unshipped.txt @nohwnd @MarcoRossignoli
+
+# Adding/removing telemetry or doing any kind of changes to the telemetry data constants is
+# something that should be treated with care and we should be transparent about it. It also
+# requires making sure all telemetry collection systems in place will continue to function well.
+TelemetryDataConstants.cs @cvpoienaru
```

```suggestion
TelemetryDataConstants.cs @cvpoienaru @nohwnd
```

Add me as well in case you are  on vacation?

---

Good point, thanks. Fixed !